server/core/src/miscServerFunct.cpp

```diff
@@ -2983,12 +2982,24 @@ irods::error get_script_output_single_line(
     const std::string&              script_name,
     const std::vector<std::string>& args,
     std::string&                    output ) {
-    bp::ipstream output_stream;
-    bp::child c(script_language, args, bp::std_out > output_stream);
-    output.clear();
-    std::getline(output_stream, output);
-    c.wait();
-    return SUCCESS();
+    namespace bp = boost::process;
+    try {
+        auto tmp = args;
+        tmp.insert(tmp.begin(), fmt::format("{}/scripts/{}", irods::get_irods_home_directory().string(), script_name));
+        bp::ipstream output_stream;
+        bp::child c(script_language, tmp, bp::std_out > output_stream);
+        output.clear();
+        if (!output_stream) {
+            return ERROR(FILE_READ_ERR, "Bad stream");
```

Comment:
The error message needs to include more information about what happened so that the admin can determine what to do.

`FILE_READ_ERR` doesn't express what happened very well. We should look for a different error code or create a new one.